app/views/welcome/index.html.erb

```diff
@@ -4,4 +4,4 @@
 
 
 <div id="MyClockDisplay" class="clock" onload="showTime()"></div>
-<%= javascript_pack_tag 'index', 'data-turbolinks-track': 'reload' %>
+<%#= javascript_pack_tag 'index', 'data-turbolinks-track': 'reload' %>
```

Comment:
If you don't occupy this line, delete it.

Reply:
This line is need it.